src/RadhydroPulseMGconst/test_radhydro_pulse_MG_const_kappa.cpp

```diff
@@ -113,17 +113,9 @@ template <> struct RadSystem_Traits<SGProblem> {
 	static constexpr int beta_order = 1;
 };
 
-template <>
-AMREX_GPU_HOST_DEVICE auto RadSystem<SGProblem>::ComputePlanckOpacity(const double /*rho*/, const double /*Tgas*/) -> quokka::valarray<double, nGroups_>
-{
-	quokka::valarray<double, nGroups_> kappaPVec{};
-	for (int g = 0; g < nGroups_; ++g) {
-		kappaPVec[g] = kappa0;
-	}
-	return kappaPVec;
-}
+template <> AMREX_GPU_HOST_DEVICE auto RadSystem<SGProblem>::ComputePlanckOpacity(const double /*rho*/, const double /*Tgas*/) -> amrex::Real { return kappa0; }
```

Comment:
Why is this function used for a multi-group test? I don't understand what's going on here.

Reply:
Because SGProblem is a single-group problem. This test has two problems: a single-group problem and a multigroup problem with constant opacity. This is to confirm that MG RHD gives the same result as the single-group RHD.